tests/Aspire.Hosting.Elasticsearch.Tests/ElasticsearchFunctionalTests.cs

```diff
@@ -26,7 +26,6 @@ public class ElasticsearchFunctionalTests(ITestOutputHelper testOutputHelper)
 
     [Fact]
     [RequiresDocker]
-    [ActiveIssue("https://github.com/dotnet/aspire/issues/5821")]
```

Comment:
Let's leave these disabled tests as disabled for now. And enable them by themselves in a new PR.